lib/Mojolicious/Plugin/CGI.pm

```diff
@@ -53,7 +53,8 @@ sub register {
 }
 
 sub _child {
-  my ($c, $args, $stdin, $stdout, $stderr) = @_;
+  my ($defaults, $c, $args, $stdin, $stdout, $stderr) = @_;
```

Comment:
Granted $c is generally conventionally first, but you have $defaults before $c for sub _run, and I followed suit.